site/utils/index.ts

```diff
@@ -238,13 +238,27 @@ export const removeDuplicates = <T, K extends keyof T>(arrayToRemoveDuplicates:
         (value, index, self) => index === self.findIndex((item) => item[key] === value[key]),
     );
 
+export const removeDuplicatesBasedOnMode = <T extends Operator, K extends keyof T>(
```

Comment:
We could do with some tests around this function

Reply:
done